wicketd/tests/integration_tests/commands.rs

```diff
@@ -0,0 +1,43 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Tests for the executable commands in this repo. Most functionality is tested
+//! elsewhere, so this really just sanity checks argument parsing, bad args, and
+//! the --openapi mode.
+
+use std::path::PathBuf;
+
+use expectorate::assert_contents;
+use omicron_test_utils::dev::test_cmds::{
+    assert_exit_code, path_to_executable, run_command, EXIT_SUCCESS,
+};
+use openapiv3::OpenAPI;
+use subprocess::Exec;
+
+// name of mgs executable
```

Comment:
```suggestion
// name of wicketd executable
```

(or remove the comment altogether; doesn't seem super valuable)